src/app/components/LinkedData/index.jsx

```diff
@@ -75,6 +77,11 @@ LinkedData.propTypes = {
   publishingPrinciples: string.isRequired,
   noBylinesPolicy: string.isRequired,
   logoUrl: string.isRequired,
+  about: arrayOf(shape({})),
```

Comment:
When we have a shape in proptypes, we should define the internal structure, such as:

```suggestion
  about: arrayOf(shape({
    '@type': string.isRequired,
     alternateName: string.isRequired,
     name: string.isRequired,
     sameAs: string.isRequired,
  })),
```